featurehub-solid-sdk/src/hooks/useFeature.tsx

```diff
@@ -15,8 +16,13 @@ function useFeature<T = boolean>(key: string): Accessor<T> {
   const [value, setValue] = createSignal(client().feature<T>(key).value);
 
   createEffect(
-    on(client, () => {
-      if (listenerId) client().feature<T>(key).removeListener(listenerId);
+    on(ready, () => {
+      // Being proper, we only subscribe to features when FeatureHub is ready
+      if (!ready()) return;
```

Comment:
Have to rely on self-managed `ready` signal since the `readinessListener` from the config can fire with `Readyness.NotReady` AFTER calling prior with `Readyness.Ready`. See comments in `FeatureHub.tsx` to see for a bit more comments/context